cmd/minikube/cmd/start.go

```diff
@@ -832,6 +852,25 @@ func validateCPUCount(local bool) {
 	if cpuCount < minimumCPUS && !viper.GetBool(force) {
 		exit.UsageT("Requested cpu count {{.requested_cpus}} is less than the minimum allowed of {{.minimum_cpus}}", out.V{"requested_cpus": cpuCount, "minimum_cpus": minimumCPUS})
 	}
+
+	if driver.IsKIC((drvName)) {
+		si, err := cachedKicSystemInfo(drvName)
+		if err != nil {
+			out.WarningT("Failed to verify '{{.driver_name}} info', ensure your {{.driver_name}} is running healthy.", out.V{"driver_namee": drvName})
+		}
```

Comment:
add a `return nil` here, as accessing `si.CPUs` may not work if an error was encountered.